src/DistributedMetadata/DDLService.cpp

```diff
@@ -245,11 +245,17 @@ Int32 DDLService::doDDL(
         auto [response, http_code] = sendRequest(uri, method, query_id, user, password, payload, log);
 
         err = toErrorCode(http_code, response);
-        if (err == ErrorCodes::OK || err == ErrorCodes::UNRETRIABLE_ERROR)
+        if (err == ErrorCodes::OK)
         {
             return err;
         }
 
+        if (err == ErrorCodes::UNRETRIABLE_ERROR)
```

Comment:
else if ?

Reply:
`else` is not need here as it  has already called `return` in the `if` above